tests/testcases/030_check-network.yml

```diff
@@ -25,6 +33,9 @@
     failed_when: false
     no_log: true
 
+  - debug: msg="{{get_pods.stdout.split('\n')}}"
```

Comment:
this needs to be moved after `- name: Get pod names`

and then use this:

`- debug: msg="{{pods.stdout.split('\n')}}"`